tests/unit/test_solvers.py

```diff
@@ -428,6 +438,146 @@ def solver_factory(self) -> jtu.Partial:
         return jtu.Partial(SteinThinning, coreset_size=coreset_size, kernel=kernel)
 
 
+class TestGreedyKernelInducingPoints(RefinementSolverTest, ExplicitSizeSolverTest):
```

Comment:
Should override `check_solution_invariants` to also check that if `unique=True`, the points are actually unique.

Reply:
Done